### PR TITLE
Migration to c2

### PR DESCRIPTION
**Pull request by astroline**
**Is this a bug fix or adding new feature?**

**What is this PR about? / Why do we need it?**

**What testing is done?**

### CODE REVIEW

File: pkg/cloud/cloud.go [hunk 1]
```diff
@@ -347,7 +383,7 @@ func (c *cloud) AttachDisk(ctx context.Context, volumeID, nodeID string) (string
 
 	if !device.IsAlreadyAssigned {
 		request := &ec2.AttachVolumeInput{
-			Device:     aws.String(device.Path),
+			Device:     aws.String("disk2"),
```
Comment (ancient07) [inline pkg/cloud/cloud.go:380]: Is it expected here?

Comment (astroline) [reply pkg/cloud/cloud.go:380]: This field means nothing for c2 api, but has to be in format "disk{number}". I will probably fix this to be incremental depending on number of disks currently attached to vm

File: deploy/kubernetes/base/controller.yaml [hunk 2]
```diff
@@ -33,6 +33,10 @@ spec:
           env:
             - name: CSI_ENDPOINT
               value: unix:///var/lib/csi/sockets/pluginproxy/csi.sock
+            - name: AWS_EC2_ENDPOINT
+              value: https://api.cloud.croc.ru
+            - name: AWS_REGION
+              value: ru-msk
```
Comment (Ubun1) [inline deploy/kubernetes/base/controller.yaml]: maybe ```croc``` ?

File: deploy/kubernetes/base/node.yaml [hunk 3]
```diff
@@ -24,14 +24,16 @@ spec:
...
+          image: dhub.c2.croc.ru/kaas/aws-ebs-csi-driver:latest
           args:
             - --endpoint=$(CSI_ENDPOINT)
             - --logtostderr
             - --v=5
           env:
             - name: CSI_ENDPOINT
               value: unix:/csi/csi.sock
+            - name: AWS_REGION
+              value: ru-msk
```
Comment (Ubun1) [inline deploy/kubernetes/base/node.yaml]: maybe ```croc``` ?

File: pkg/cloud/cloud.go [hunk 4]
```diff
@@ -38,22 +41,29 @@ import (
 const (
 	// VolumeTypeIO1 represents a provisioned IOPS SSD type of volume.
 	VolumeTypeIO1 = "io1"
+	// VolumeTypeIO2 represents a provisioned IOPS SSD type of volume.
+	VolumeTypeIO2 = "io2"
 	// VolumeTypeGP2 represents a general purpose SSD type of volume.
 	VolumeTypeGP2 = "gp2"
 	// VolumeTypeSC1 represents a cold HDD (sc1) type of volume.
 	VolumeTypeSC1 = "sc1"
```
Comment (Ubun1) [inline pkg/cloud/cloud.go]: there is no such volume type in croc cloud

File: pkg/cloud/cloud.go [hunk 5]
```diff
@@ -38,22 +41,29 @@ import (
...
 	// VolumeTypeST1 represents a throughput-optimized HDD type of volume.
 	VolumeTypeST1 = "st1"
```
Comment (Ubun1) [inline pkg/cloud/cloud.go]: there is no such volume type in croc cloud

File: pkg/cloud/devicemanager/manager.go [hunk 6]
```diff
@@ -175,12 +166,12 @@ func (d *deviceManager) release(device *Device) error {
 	d.mux.Lock()
 	defer d.mux.Unlock()
 
-	var name string
+	/*var name string
```
Comment (Ubun1) [inline pkg/cloud/devicemanager/manager.go]: if you dont need this code, lets remove it

**Review by Ubun1 — CHANGES_REQUESTED** [review]

**Review by Ubun1 — APPROVED** [review]

**Pull request merged**